packages/forma-36-react-components/src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import * as CSS from 'csstype';
+import tokens from '@contentful/forma-36-tokens';
+import cn from 'classnames';
+
+import styles from './Grid.css';
+
+export type GapTypes = 'none' | 'spacing2xs' | 'spacingXs' | 'spacingS' | 'spacingM' | 'spacingL' | 'spacingXl' | 'spacing2Xl' | 'spacing3Xl' | 'spacing4Xl';
+
+export interface GridProps {
+  /**
+   * Class names to be appended to the className prop of the component */
+  className?: string;
+  /**
+   * Child nodes to be rendered in the component */
+  children?: React.ReactNode;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id) */
+  testId?: string;
+  /**
+   * Defines how many columns, default is `auto` */
+  columns?: number | CSS.GridTemplateColumnsProperty<string>;
+  /**
+   * Defines how many rows, default is `auto` */
+  rows?: number | CSS.GridTemplateColumnsProperty<string>;
+  /**
+   * Spaces between rows, corresponds to of spacing tokens values, default is none */
+  rowGap?: GapTypes;
+  /**
+   * One of Spacing tokens values, default is 0 */
+  columnGap?: GapTypes;
+  /**
+   * One of grid-auto-flow css values */
+  flow?: CSS.GridAutoFlowProperty
+  /**
+   * Sets display:inline-grid */
+  inline?: boolean;
+  /**
+   * One of justify-content css values */
+  justifyContent?: CSS.JustifyContentProperty;
+  /**
+   * One of justify-content css values */
+  alignContent?: CSS.AlignContentProperty;
+  /**
+   * style prop, for inline styles */
+  style?: React.CSSProperties;
+}
+
+const defaultProps = {
+  columns: 'auto',
+  rows: 'auto',
+  columnGap: 'spacingM',
+  rowGap: 'none',
+  testId: 'cf-ui-grid',
+};
+
+export const Grid = (props: GridProps) => {
```

Comment:
I saw so many class components in Forma that I thought we had to use class for some reason 🙄 

I'm glad to know that I was wrong hahaha... I think I will refactor my Table enhancement

Reply:
yeah, I agree we should change these class components whenever we get the chance. 👍